apps/beeswax/src/beeswax/server/hive_server2_lib_tests.py

```diff
@@ -92,8 +92,8 @@ def test_open_session(self):
         Session.objects.filter(owner=self.user, application=self.query_server['server_name']).count()
       )
       assert_equal(
-        session.guid,
-        Session.objects.get_session(self.user, self.query_server['server_name']).guid.encode()
+        str(session.guid),
```

Comment:
Does 'hue test specific ' of this test fails too?

Reply:
yes, 
<img width="1499" alt="Screenshot 2021-01-12 at 11 40 32 PM" src="https://user-images.githubusercontent.com/36241930/104354689-94034c00-552f-11eb-8d79-2b01238f6cd9.png">

Reply:
So not sure why, probably the behavior of `TextField`  changed, but +1 for the change